main.go

```diff
@@ -122,6 +122,9 @@ func run(ctx context.Context) error {
 
 	router.StrictSlash(true).Path("/datasets/{datasetID}/editions/{editionID}/versions/{versionID}/filter").Methods("POST").HandlerFunc(handlers.CreateFilterID(f, dc, *cfg))
 
+	// TODO: fix this path
+	router.StrictSlash(true).Path("/economy/inflationandpriceindices/datasets/understandingthedifferentapproachesofmeasuringowneroccupiershousingcosts/current").Methods("GET").HandlerFunc(handlers.DatasetPage(zc, dc, rend, *cfg, apiRouterVersion))
```

Comment:
@mr-nick17

So am I right in thinking this is related to the "hack" everyone is talking about where the new template that was made is only going to work for this particular url/dataset?

Reply:
fixed to use correct code for routing